packages/cozy-codemods/src/transforms/__testfixtures__/replace-icons-svgr.output.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import CustomIcon from './CustomIcon'
+
+import MagnifierIcon from "cozy-ui/transpiled/react/Icons/Magnifier";
```

Comment:
Can we have in output? If not, not a big deal since our linter will fix it. 

```jsx
import MagnifierIcon from 'cozy-ui/transpiled/react/Icons/Magnifier'
```

Reply:
I don't see how to easily lint inside the transform runner how jest. I think I wrote in the commit that indeed, it was not veyr important to have semicolons in the output since we run our eslint afterwards.